src/config/config.ts

```diff
@@ -148,6 +148,10 @@ export class ServerConfigHelpers {
         return config.serverConfig.sessionServiceEnabled;
     }
 
+    static oncoKbIsEnabledWithoutToken() {
+        return config.serverConfig.show_oncokb && !config.serverConfig.oncoKbTokenDefined;
```

Comment:
Isn't it odd that we use different formats for the variables under serverConfig? @onursumer

Reply:
definitely! but this needs to be fixed on the back end.